lstchain/image/tests/test_modifier.py

```diff
@@ -77,11 +82,6 @@ def test_calculate_required_additional_nsb(mc_gamma_testfile, observed_dl1_files
 
 
 def test_tune_nsb_on_waveform():
-    import os
-    import astropy.units as u
-    from scipy.interpolate import interp1d
-    from lstchain.image.modifier import tune_nsb_on_waveform
```

Comment:
It is good practice to import the internal imports here. That way import errors appear as part of the test and do not break the whole test suite